acceptance/acceptance_test.go

```diff
@@ -108,6 +108,14 @@ func testAccept(t *testing.T, inprocessMode bool, singleTest string) int {
 	// Consistent behavior of locale-dependent tools, such as 'sort'
 	t.Setenv("LC_ALL", "C")
 
+	// MSYS2 automatically converts absolute paths like /Users/$username/$UNIQUE_NAME to
+	// C:/Program Files/Git/Users/$username/UNIQUE_NAME before passing it to the CLI
+	// Setting this environment variable prevents that conversion on windows.
+	// This is fine to do globally because:
+	// 1. In most contexts an absolute path refers to a file in a Databricks workspace.
+	// 2. Most if not all tests will use relative paths for local files.
+	t.Setenv("MSYS_NO_PATHCONV", "1")
```

Comment:
It's better to use Env setting in test.toml to set env var. This makes it overridable in subtests.

Reply:
I do wonder if we need it globally though. Perhaps our users don't set it / not aware of it (I wasn't), so maybe it's good to know when it's needed and maybe add some protection in CLI against unwanted conversion?

Reply:
It does look like a common recommendation to set that env var on that platform to prevent mangling https://www.google.com/search?q=MSYS_NO_PATHCONV

So I guess we can recommend it as well.

Would be nice to detect the mangling (e.g. remote path contains ":/"), not sure if we do that already.

Reply:
> So I guess we can recommend it as well.

Do you mean to the users of the CLI itself outside the context of acceptance tests?

Reply:
Exactly. My point is -- it's not just a problem with acceptance tests, it's a problem for our users on MSYS2.

Reply:
True, but users on MSYS2 would know about this quirk given they use it regularly. Given no customers have actively complained about this to us I don't think we need to actively prioritize detection of MSYS2.

Reply:
NVM, a customer just complained: https://github.com/databricks/cli/issues/2701

Reply:
Thanks, moved the env var override to test.toml